pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -404,6 +401,13 @@ self: super: ({
   # https://github.com/NixOS/nixpkgs/issues/149692
   Agda = disableCabalFlag "optimise-heavily" super.Agda;
 
+} // lib.optionalAttrs (pkgs.stdenv.isAarch64 && super.ghc.version == "9.0.2") {
```

Comment:
I seem to recall this also affected x86_64?

Reply:
https://github.com/kazu-yamamoto/crypton/issues/35#issuecomment-2081407456